src/ripple/app/tx/impl/InvariantCheck.cpp

```diff
@@ -815,6 +815,10 @@ ValidMPTIssuance::visitEntry(
             mptIssuancesDeleted_++;
         else if (!before)
             mptIssuancesCreated_++;
+
+        if ((*after)[sfOutstandingAmount] >
```

Comment:
This is checked in `rippleMPTCredit`. Why is it needed as invariant?

Reply:
I think it's cheap and wouldn't hurt to check

Reply:
Well, but it's completely redundant, right? This is why we have the transactor's checks. If "cheap and wouldn't hurt" argument is followed then the invariant might have so many checks that it's no longer cheap. This is just my opinion.

Reply:
fair enough, i dont have a strong opinion on this. Im happy with removing it

Reply:
just pushed a commit to remove it